Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/dialogs/TaskDialog.kt

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+package org.dolphinemu.dolphinemu.dialogs
+
+import android.app.Dialog
+import android.content.DialogInterface
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import androidx.lifecycle.ViewModelProvider
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import org.dolphinemu.dolphinemu.R
+import org.dolphinemu.dolphinemu.model.TaskViewModel
+
+class TaskDialog : DialogFragment() {
+    private lateinit var viewModel: TaskViewModel
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        viewModel = ViewModelProvider(requireActivity())[TaskViewModel::class.java]
+
+        val dialogBuilder = MaterialAlertDialogBuilder(requireContext())
+            .setTitle(requireArguments().getInt(KEY_TITLE))
+            .setView(R.layout.dialog_indeterminate_progress)
+        if (requireArguments().getBoolean(KEY_CANCELLABLE)) {
+            dialogBuilder.setCancelable(false)
```

Comment:
Why are you calling `setCancelable(false)` only when `KEY_CANCELLABLE` is `true`?

Reply:
My bad